src/index.css

```diff
@@ -19,5 +23,59 @@
   nav a.active {
     @apply bg-gray-100
   }
+
+  .standard-table {
```

Comment:
rename to comfortable-table + add styles for a compact table (with a button group selector for switching them)

Reply:
![image](https://user-images.githubusercontent.com/20907048/153588622-82bba6fd-ca5d-4b32-8a05-54ae0eda5cca.png)
1. is this "comptact table"?
2. is this button group selector for demo purposes, or am i missing something?

Reply:
done